tests/test_Source/test_quasar.py

```diff
@@ -1,7 +1,7 @@
 from astropy.cosmology import FlatLambdaCDM
 from astropy.units import Quantity
 from slsim.Sources.quasars import Quasars
-from slsim.Sources.quasar_catalog.simple_quasar import quasar_catalog_simple
+from slsim.Sources.Quasar_catalog.simple_quasar import quasar_catalog_simple
```

Comment:
naming convention: A folder I would set as QuasarCatalog instead, same conventions as classes vs definitions for folders vs files